packages/common/src/types/errors.ts

```diff
@@ -123,6 +123,7 @@ export class UnexpectedServerError extends LightdashError {
         message = 'Something went wrong.',
         data: { [key: string]: any } = {},
     ) {
+        console.log(message, data);
```

Comment:
Just the log 🫣

Reply:
oh good catch. altho, a very necessary log 🤣